docker/builder.dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM python:2.7.16-slim-buster as sdist
+
+LABEL maintainer="oss@sentry.io"
+LABEL org.opencontainers.image.title="Sentry Wheel Builder"
+LABEL org.opencontainers.image.description="Python Wheel Builder for Sentry"
+LABEL org.opencontainers.image.url="https://sentry.io/"
+LABEL org.opencontainers.image.source="https://github.com/getsentry/sentry"
+LABEL org.opencontainers.image.vendor="Functional Software, Inc."
+LABEL org.opencontainers.image.authors="oss@sentry.io"
+
+# Sane defaults for pip
+ENV PIP_NO_CACHE_DIR=off \
+  PIP_DISABLE_PIP_VERSION_CHECK=1
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  # Needed for fetching stuff
+  wget \
+  && rm -rf /var/lib/apt/lists/* \
+  # Needed to extract final dependencies from the whl
+  && pip install pkginfo==1.5.0.1
+
+# Get and set up Node for front-end asset building
+ENV VOLTA_VERSION=0.8.0 \
```

Comment:
[Volta 0.8.1](https://github.com/volta-cli/volta/releases/tag/v0.8.1) (just released!) shaved off over 2 MB from the binary, well worth bumping.